README.md

```diff
@@ -401,7 +444,7 @@ The `iotedgedev` module has the following commands:
 - `--show-logs`          Opens a new terminal window for edgeAgent, edgeHub and each edge module. You can configure the terminal command with LOGS_CMD.
 - `--save-logs`          Saves edgeAgent, edgeHub and each edge module logs to LOGS_PATH.
 
-**iotedgedev commands**
+iotedgedev azure**iotedgedev commands**
```

Comment:
This line looks funky: 

![image](https://user-images.githubusercontent.com/2163001/35869280-8a65baf2-0b13-11e8-8f53-5e23d38a4565.png)


I use http-server npm package to run server in root of repo, then localhost:8080/readme.md

and then in chrome I test with: https://chrome.google.com/webstore/detail/github-flavored-markdown/faelggnmhofdamhdegcdhhemfokkfngk